recipes/mumps/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ `uname` == "Darwin" ]; then
+  cp $RECIPE_DIR/Makefile.debian.SEQ_mac Makefile.inc
+  export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+  cp $RECIPE_DIR/Makefile.debian.SEQ Makefile.inc
```

Comment:
Yeah, Linux seems to get away without having to do this sort of thing sometimes, but I figure just being careful is cheap.